src/components/DataDisplay/DataTable/DataTable.tsx

```diff
@@ -226,7 +264,7 @@ const DataTableCell = forwardRef<HTMLTableCellElement, DataTableCellProps>(
         className={twMerge(
           "whitespace-nowrap p-4",
           (selected || color === "primary") &&
-            "text-primary-600 dark:text-primary-400",
+            "font-bold text-primary-600 dark:text-primary-400",
```

Comment:
Just want to make sure by bolding this cell we dont cause any extra layout shifting due to increased text size.

Reply:
It's possible that it might. But more often than not, that feature is paired with scrolling behaviors. That's going to cover visual shift _within_ the table easily. Affecting surrounding layout... That'd be a question for implementation. The scroll behaviors only work if it's x and y constrained so I expect it should be safe. If it's free layout, I see rare use cases to use selected other than statically when rendered.